addons/ui/org.openhab.ui.cometvisu/ESH-INF/config/config.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<config-description:config-descriptions xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:config-description="http://eclipse.org/smarthome/schemas/config-description/v1.0.0"
+	xsi:schemaLocation="http://eclipse.org/smarthome/schemas/config-description/v1.0.0
+        http://eclipse.org/smarthome/schemas/config-description-1.0.0.xsd">
+
+	<config-description uri="ui:cometvisu">
+		<parameter name="webFolder" type="text">
+            <label>Filesystem path</label>
+            <description>Defines the absolute path to the CometVisu client directory.</description>
+            <default>userdata/cometvisu</default>
```

Comment:
I would remove the default here. Firstly, it is NOT an absolute path (although the description says it has to be) and secondly, the path is a different one on apt installations.